src/config/scratchpad.rs

```diff
@@ -4,4 +4,29 @@ use serde::{Deserialize, Serialize};
 pub struct ScratchPad {
     pub name: String,
     pub value: String,
+    // relative width of scratchpad, 50 means 50% of workspace width
+    #[serde(default)]
+    pub width: i32,
+    // relative height of scratchpad, 50 means 50% of workspace height
+    #[serde(default)]
+    pub height: i32,
+    // relative x of scratchpad, 25 means 25% of workspace x
+    #[serde(default)]
+    pub x: i32,
+    // relative y of scratchpad, 25 means 25% of workspace y
+    #[serde(default)]
+    pub y: i32,
+}
+
+impl Default for ScratchPad {
+    fn default() -> Self {
+        Self {
+            name: "st".to_string(),
+            value: "st".to_string(),
```

Comment:
Should `st` really be the default?

Reply:
That's a default which was not called before. Actually without this could be better. When no scratch pad configured, simply don't show it. What do you think?